DevCycle.SDK.Server.Cloud/Api/DevCycleCloudClient.cs

```diff
@@ -22,67 +19,75 @@ public override DevCycleCloudClient Build()
             return new DevCycleCloudClient(sdkKey, loggerFactory, options, restClientOptions);
         }
     }
+
     public sealed class DevCycleCloudClient : DevCycleBaseClient
     {
         private readonly DevCycleApiClient apiClient;
         private readonly ILogger logger;
-
+        private DevCycleProvider OpenFeatureProvider { get; }
         private readonly DevCycleCloudOptions options;
 
         internal DevCycleCloudClient(
             string sdkKey,
-            ILoggerFactory loggerFactory, 
-            IDevCycleOptions options=null,
+            ILoggerFactory loggerFactory,
+            IDevCycleOptions options = null,
             DevCycleRestClientOptions restClientOptions = null
-        ) {
+        )
+        {
             ValidateSDKKey(sdkKey);
             apiClient = new DevCycleApiClient(sdkKey, restClientOptions);
             logger = loggerFactory.CreateLogger<DevCycleCloudClient>();
-            this.options = options != null ? (DevCycleCloudOptions) options : new DevCycleCloudOptions();
+            this.options = options != null ? (DevCycleCloudOptions)options : new DevCycleCloudOptions();
+            OpenFeatureProvider = new DevCycleProvider(this);
```

Comment:
I wonder if we should only create this if it's requested.

Reply:
It's a static class - so there's no real memory footprint since it's a wrapper re-referencing the client itself. We can convert it to init only on request - but that would in theory slow down the initialization of the OF provider.